playground/mocks/config/responseConfig.js

```diff
@@ -53,12 +53,11 @@ const idx = {
     // 'identify-with-universal-link',
     // 'select-factor-authenticate',
     // 'select-factor-for-password-recovery',
-    // 'success',
+    'success',
```

Comment:
We should just always  default here to `identify` response. Every time I pull the latest and run `yarn start` I see a different screen that loads up 😆 .

Reply:
😅